README.md

```diff
@@ -1,43 +1,37 @@
 
 ## Why another language learning?
 
-All language learning platform feed user by thier own data, that may cover lots of unknown but it definetly is not working in user context.
-
-- How many times did you face a new word while you are reading?
-
-### 😿 story
-
-Ofcourse you can search for the meaning, and save the word into your short term memory (and keep it for 6 sec), but we all know process of transfering a word from your mind-langauage to daily speek is not this easy.
-
-### Moose will be your 🚀
+1) We want to focus on user provided data rather that our data. 
+What does "user provided data" it mean? Simply user provide their own words and we provide tools to help him succesfuly learn it. Most language learning platform feed user by thier own data, that may cover lots of unknown but it definetly is not working in user context.
+> How many times did you face a new word while you are reading?
 
-Here is were we think we have a new idea 👍🏻
+2) We want to make learning new words fun!
+Our goal is to provide tools to make learning new vocabulary easier and entertaining at the same time. Our app is not for everyone but we want to make it enjoyable for everyone!
+> We recreate some old minigames and even create new minigames.
 
-Nowadyas every one knows that a words must be memorized in a context.
-**parabellum** means a thousnad time more when you saw [John Wick 3 - Parabellum](https://en.wikipedia.org/wiki/John_Wick:_Chapter_3_%E2%80%93_Parabellum)
+3)Here is were we think we have a new idea 👍🏻
+Nowadyas every one knows that a words must be memorized in a context. Our Jobs is to bind and connect context for you. Wait a second 🤠, this is what all other apps do too, so what's the point?
+> This is our secret sauce.
 
-> Note: parabellum means `prepare for war`
+## About project and the team 😄 
 
-So should I watch movie for every word?
+"Svět" means (world) in Czech language yâd means (thought, idea) in Persian.
 
-> really 😐
+Our team is made of 3 friends. We are just a geeks who want to have fun and make learning new vocabulary more enjoyable. One of us is from Iran and the other twi is from Czech Republic.
 
-Our Jobs is to bind and connect context for you.
+## We have our own mascot and he have lots of buddies! 
 
-Wait a second 🤠, this is what all other apps do too, so what's the point?
+<img src="" alt="" align="center"/>
+> His is Moose and this animal represent headstrong, unstoppable, longevity! 
 
-> This is the secrete sauce which you must know in our meetings.
 
-## feel free to send your request, idea, interest, enthustast via
+## Feel free to send your request, idea, interest, enthustast via
 
-[Telegram](https://t.me/svetyad)
-or [GitHub discussion](https://github.com/amirHossein-Ebrahimi/Svet-yad/discussions)
+Or even if you want to join us. Send us an message!
+[Telegram](https://t.me/svetyad) or [GitHub discussion](https://github.com/amirHossein-Ebrahimi/Svet-yad/discussions)
```

Comment:
could you remove svetyad telegram channel, we maid that private for our personal project management now